src/currencyConversion/ExchangeRateAPI.ts

```
@@ -33,19 +34,30 @@ export default class ExchangeRateAPI extends RESTDataSource {
     sourceCurrencyCode,
     currencyCode,
     timestamp,
-  }: CurrencyConversionArgs): Promise<BigNumber> {
+  }: CurrencyConversionArgs): Promise<BigNumber | undefined> {
     if (!currencyCode) {
       throw new Error('No currency code specified')
     }
 
-    const date = timestamp ? new Date(timestamp) : new Date()
-    const fetchedRate = await this.queryExchangeRate(
-      sourceCurrencyCode || USD,
-      currencyCode,
-      date,
-    )
+    try {
+      const date = timestamp ? new Date(timestamp) : new Date()
+      const fetchedRate = await this.queryExchangeRate(
+        sourceCurrencyCode || USD,
+        currencyCode,
+        date,
+      )
 
-    return new BigNumber(fetchedRate)
+      return new BigNumber(fetchedRate)
+    } catch (error) {
+      logger.error({
```

Can you add a comment here on why it's better to log the error, suppress it, and return `undefined` than let the error bubble up to the ApolloServer (and have the default logger log it)?